docs/dev/README.md

```diff
@@ -0,0 +1,269 @@
+# Tekton Dashboard - Dev docs
+
+- Running tests
+- Troubleshooting
+
+This guide explains how to build, deploy and test the Tekton Dashboard. It covers the following topics:
+
+- [Before you begin](#before-you-begin)
+- [Pre-requisites](#pre-requisites)
+- [Checkout your fork](#checkout-your-fork)
+- [Build the frontend](#build-the-frontend)
+- [Build the backend](#build-the-backend)
+  - [Command line arguments](#command-line-arguments)
+- [Build YAML manifests](#build-yaml-manifests)
+- [Build and deploy with kustomize and ko](#build-and-deploy-with-kustomize-and-ko)
+  - [Build docker image for another GOARCH](#build-docker-image-for-another-goarch)
+- [Development server](#development-server)
+- [Run backend tests](#run-backend-tests)
+- [Run frontend tests](#run-backend-tests)
+  - [Unit tests](#unit-tests)
+  - [Linter](#linter)
+- [i18n](#i18n)
+- [Storybook](#storybook)
+- [Troubleshooting](#troubleshooting)
+- [Next steps](#next-steps)
+
+## Before you begin
+
+Many of the instructions here replicate what's in the [tektoncd/pipeline development guide](https://github.com/tektoncd/pipeline/blob/master/DEVELOPMENT.md), with a couple of caveats currently.
+
+- This project has not yet been tested with GKE
```

Comment:
The dogfooding and robocat clusters are on GKE. I think we can just remove this line.

Reply:
was not sure, copied from the old docs